apps/dashboard/src/components/engine/status/components/ErrorRate.tsx

```diff
@@ -0,0 +1,112 @@
+"use client";
+
+import { useQuery } from "@tanstack/react-query";
+import { Area, AreaChart, CartesianGrid, XAxis, YAxis } from "recharts";
+
+import {
+  Card,
+  CardContent,
+  CardDescription,
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+import {
+  type ChartConfig,
+  ChartContainer,
+  ChartLegend,
+  ChartLegendContent,
+  ChartTooltip,
+  ChartTooltipContent,
+} from "@/components/ui/chart";
+
+export function ErrorRate() {
+  const monitorData = useQuery(["errorRate"], async () => {
```

Comment:
please use object notation for useQuery:

```
useQuery({
queryKey: [....]
queryFn: () => ...
});
```

also `errorRate` is not a good enough query key

since this is unique per instance the query key _has_ to include the engine instance id / url (something unique) and probably _should_ include that we're talking about engine data here, suggestion:

`["engine", <instance.id>, "errorRate"]`